src/index.html

```diff
@@ -5,8 +5,437 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home">
+        <img
+          class="header__logo"
+          src="images/logo.png"
+          alt="MOYO"
+        >
+      </a>
+      <nav>
+        <ul class="header__list">
+          <li class="header__item">
```

Comment:
nav should be a separate bem block, create `nav` block and elements from this block